### PR TITLE
Added decision-log and members

### PR DESCRIPTION
**Pull request by dgarcia360**

### CODE REVIEW

File: governance/README.md [hunk 1]
```diff
@@ -32,4 +32,20 @@ TODO: more to be flushed out here for forming and end of working groups, idea of
 
 # Project Mangement Committee (PMC)
 
-The PMC is the top level group for managing the project, which was seeded with the original "3 core devs".  The job of the PMC is to be the stewards of the project, community and its software components, irrespective of which commerical or not-for-profit organization they work with or for.  Where as SIGs and Working groups happen more in the open, on community communication channels, the PMC typically operates behind the scenes working with chairs of SIGs and voting on bigger project and community decisions.
\ No newline at end of file
+The PMC is the top level group for managing the project, which was seeded with the original "3 core devs".  The job of the PMC is to be the stewards of the project, community and its software components, irrespective of which commerical or not-for-profit organization they work with or for.  Where as SIGs and Working groups happen more in the open, on community communication channels, the PMC typically operates behind the scenes working with chairs of SIGs and voting on bigger project and community decisions.
```
Comment (Jaguar0625) [inline governance/README.md]: 1. not really community, PMC is more tech-focused imo
2. they work with or for => for whom they work
3. Whereas

File: governance/README.md [hunk 2]
```diff
@@ -32,4 +32,20 @@ TODO: more to be flushed out here for forming and end of working groups, idea of
...
+| @BloodyRookie |                      | 1 Jan 2019     |
+| @gimre        |                      | 1 Jan 2019     |
+| @kaiyzen      |                      | 1 Jan 2019     |
+| @daoka        |                      | 8 Jan 2019     |
+| @dgarcia360   | david@nem.foundation | 8 Jan 2019     |
+| @evias        |                      | 31 Jan 2020    |
+| @44uk         |                      | 6 Feb 2020     |
+| @wayonb       |                      | 6 Feb 2020     |
+| @rg911        |                      | 6 Feb 2020     |
+| @CrackTheCode |                      | 6 Feb 2020     |
```
Comment (Jaguar0625) [inline governance/README.md]: add roger

File: governance/decision-log.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Decision Log
+
+Decisions and consultations proposed and voted by the PMC.
+
+| Proposal | Status | Proposed by | Decision Date | Comment |
+|-----------------------------------------------------|-------------|---------------|---------------|-----|
+| Move nemtech org on GitHub to symbolplatform        | CONSULTATION| @dgarcia360   | 20 Feb 2020   | - keep it as nemtech (5 votes) <br> - move org to symbolplatform account (3 votes) |
+| Opinion on finalization                             | CONSULTATION| @Jaguar0625  | 14 Feb 2020    | -  not required for public launch but should be added immediately after (6 votes) <br> - required for public launch (4 votes) <br> - not required for public launch but should be added "eventually" (3 votes) |
```
Comment (Jaguar0625) [inline governance/decision-log.md]: you might what some commentary why number of votes > number of PMC members

File: governance/README.md (hunk 2 above)
Comment (Jaguar0625) [reply governance/README.md]: also in decision-log

File: governance/README.md (hunk 1 above)
Comment (evias) [reply governance/README.md]: Typo commercial

**Comment by dgarcia360** [comment]
@Jaguar0625 Solved, please review again.

**Pull request merged**